nextjs-app/components/common/Button/FloatingButtons.tsx

```diff
@@ -0,0 +1,38 @@
+import RegisterSailorButton from "./RegisterSailorButton";
+import RegisterNavigatorButton from "./RegisterNavigatorButton";
+import { twMerge } from "tailwind-merge";
+
+const FloatingButtons = () => {
+  return (
+    <>
+      <div
+        className={twMerge(
+          "fixed bottom-[144px] -right-[160px] inline-flex flex-col space-y-3 z-50",
+          "hover:right-0 transition-all duration-300"
+        )}
+      >
+        <RegisterSailorButton
+          className={twMerge(
+            "hover:bg-blue-8 hover:border-neutral-8",
+            "[&>*]:hover:text-white [&>span]:hover:border-white"
+          )}
+        />
```

Comment:
```suggestion
        <RegisterSailorButton
          className={twMerge(
            "hover:bg-blue-8 hover:border-neutral-8",
            "[&>*]:hover:text-white [&>span]:hover:border-white hover:transition duration-300"
          )}
        />
```

Should we add transition for hover ?
Here is the different

https://github.com/user-attachments/assets/fccaea27-e400-4f32-8c21-63c63220a7da